index.js

```diff
@@ -36,7 +36,7 @@ module.exports = function (path, opts) {
   // default extension to `html`
   if (!opts.default) opts.default = 'html';
 
-  debug(fmt('options: %s', opts));
+  debug(fmt('options: ', opts));
```

Comment:
whichever you prefer :) mine version accepts anything, not sure if that is anything extra useful/harmful though